models/action.go

```diff
@@ -633,7 +632,12 @@ func CommitRepoAction(opts CommitRepoActionOptions) error {
 			})
 		}
 
+	case ActionDeleteBranch: // Delete Branch
+		isHookEventPush = true
+
 	case ActionPushTag: // Create
+		isHookEventPush = true
```

Comment:
This assignment has no effect, since if you reach here you will return on line 649.

Reply:
Same as above.